swimlane/core/fields/history.py

```diff
@@ -66,6 +64,31 @@ def for_json(self):
             'user': self.user.for_json()
         }
 
+    @property
+    def app_revision(self):
+        """Deferring request for app revision until needed"""
+        if not self.__app_revision:
+            # Avoid circular imports
+            from swimlane.core.resources.app import App
+
+            app_revision_number = int(self._raw['version']['applicationRevision'])
+            app_revision_raw = self._swimlane.request(
+                'get',
+                '/app/{}/history/{}'.format(
+                    self.record.app.id,
+                    app_revision_number)).json()
+            self.__app_revision = App(self._swimlane, app_revision_raw['version'])
```

Comment:
`app_revision_raw` is a list and not a dict so it's expecting an int position instead of a property name.